src/AgentImplementation.sol

```diff
@@ -62,11 +62,6 @@ contract AgentImplementation is IAgent, ERC721Holder, ERC1155Holder {
         permit2 = permit2_;
     }
 
-    /// @notice Return current fee charging status when calling
-    function isCharging() external view returns (bool) {
-        return _callbackWithCharge.isCharging();
-    }
-
```

Comment:
Which coding style do we follow for the function order? Do we follow the public order or the group order?

Public order: order functions in a file
Group order: order functions in groups

Reply:
Group order is in higher priority when it helps to classify the function into different groups for better readability. However the grouping might not be trivial so additional discussion is needed. When the contract is not too complicated, applying public order should be the most straightforward way to do the work.